octopusdeploy/resource_usernamepassword_account.go

```diff
@@ -1,149 +1,68 @@
 package octopusdeploy
 
 import (
-	"fmt"
-
-	"github.com/OctopusDeploy/go-octopusdeploy/client"
-	"github.com/OctopusDeploy/go-octopusdeploy/enum"
-	"github.com/OctopusDeploy/go-octopusdeploy/model"
+	"github.com/OctopusDeploy/go-octopusdeploy/octopusdeploy"
 	"github.com/hashicorp/terraform/helper/schema"
 )
 
 func resourceUsernamePassword() *schema.Resource {
+
+	schemaMap := getCommonAccountsSchema()
```

Comment:
This will get all the common schema fields (and any we add in the future). This means it doesn't need to be repeated for every account type.